docs/en/install-upgrade/release-notes/release-notes-kibana.asciidoc

```diff
@@ -6,3 +6,503 @@
+.Removed legacy risk engine features.
+[%collapsible]
+====
+*Details* +
+For more information, refer to ({kibana-pull}201810[#201810]).
+====
+
+//Needs to be added to upgrade notes and detailed    
+[discrete]
+[[deprecation-199598]]
+.Removed EDR Workflows deprecated APIs.
+[%collapsible]
+====
+*Details* +
+
+APIs removed:
+
+- `/api/endpoint/isolate`
+- `/api/endpoint/unisolate`
+- `/api/endpoint/policy/summaries`
+- `/api/endpoint/suggestions/{suggestion_type}`
+- `/api/endpoint/action_log/{agent_id}`
+
+For more information, refer to ({kibana-pull}199598[#199598]).
+====
+      
+[float]
+[[deprecations-9.0.0]]
+== Deprecations
+
+The following functionality is deprecated in 9.0.0, and will be removed in 10.0.0.
+Deprecated functionality does not have an immediate impact on your application, but we strongly recommend
+you make the necessary updates after you upgrade to 9.0.0.
+
+      
+[discrete]
+[[deprecation-201313]]
+.Removed ephemeral tasks from task manager, action, and alerting plugins.
+[%collapsible]
+====
+*Details* +
+
+Deprecates the following configuration settings:
+
+* `xpack.task_manager.ephemeral_tasks.enabled`
+* `xpack.task_manager.ephemeral_tasks.request_capacity`
+* `xpack.alerting.maxEphemeralActionsPerAlert`
+
+No action is required on the user's end. These settings will no longer have any effect as ephemeral tasks are now removed.
+
+For more information, refer to ({kibana-pull}201313[#201313]).
+====
+      
+   
+[discrete]
+[[deprecation-197802]]
+.Removed `visualization:colorMapping` advanced setting.
+[%collapsible]
+====
+*Details* +
+The `visualization:colorMapping` advanced setting for TSVB and Visualize charts has been removed. You can switch to Lens charts, which offer a more advanced, per-chart color mapping feature with enhanced configuration options. For more details, refer to link:https://github.com/elastic/kibana/pull/162389[#162389].
+====
+      
+
+[float]
+[[features-9.0.0]]
+== Features
+{kib} 9.0.0 adds the following new and notable features.
+
+Data ingestion and Fleet::
+* New setting allowing automatic deletion of unenrolled agents in Fleet settings ({kibana-pull}195544[#195544]).
+Elastic Security solution::
+For the Elastic Security 9.0.0 release information, refer to <<release-notes-security-9.0.0-beta1,_Elastic Security Solution Release Notes_>>.
+Kibana security::
+* New interface for Automatic Import CEL generation flow ({kibana-pull}206491[#206491]).
+
+//For more information about the features introduced in 9.0.0, refer to <<whats-new,What's new in 9.0>>.
+
+[[enhancements-and-bug-fixes-v9.0.0]]
+== Enhancements and bug fixes
+
+//For detailed information about the 9.0.0 release, review the enhancements and bug fixes.    
+
+[float]
+[[enhancement-v9.0.0]]
+=== Enhancements
+//Dashboards & Visualizations::
+Data ingestion and Fleet::
+* Improves filtering and visibility of `Uninstalled` and `Orphaned` agents in Fleet, by differentiating them from `Offline` agents.
```

Comment:
```suggestion
* Improves filtering and visibility of `Uninstalled` and `Orphaned` agents in Fleet, by differentiating them from `Offline` agents ({kibana-pull}205815[#205815]).
```
Strange that the PR number didn't get pulled in for this one!

Reply:
That's my bad I probably removed it unintentionally